app/models/invenio_rdm_record_converter.rb

```diff
@@ -63,9 +63,28 @@ def generic_file_content_path(checksum)
   end
 
   def extra_data(generic_file)
+    data = {}
+
     if !generic_file.based_near.empty?
+        data["presentation_location"] = generic_file.based_near
+    end
+    data["owner"] = owner_info(generic_file)
+
+    data
+  end
+
+  def owner_info(generic_file)
+    user = User.find_by(username: generic_file.depositor)
+
+    if user
+      {
+        "netid": user.username,
+        "email": user.email
+      }
+    else
       {
-        "presentation_location": generic_file.based_near
+        "netid": "unknown",
+        "email": "unknown"
```

Comment:
:heavy_check_mark: I will make the Prsim admin user be the owner for those.

Reply:
Okay, sounds good to me. I am pretty sure this `"unknown"` case will be unlikely. Users can't change their usernames, or change the GenericFile's depositor. But, just in case.